bach/tests/functional/bach/test_df_from.py

```diff
@@ -12,14 +12,24 @@
 from bach import DataFrame
 from sql_models.model import CustomSqlModelBuilder, SqlModel, Materialization
 from sql_models.sql_generator import to_sql
-from sql_models.util import is_postgres, is_bigquery
+from sql_models.util import is_postgres, is_bigquery, is_athena
+from tests.conftest import DB_ATHENA_LOCATION
 from tests.functional.bach.test_data_and_utils import assert_equals_data
 
-pytestmark = pytest.mark.skip_athena_todo()  # TODO: Athena
-
 def _create_test_table(engine: Engine, table_name: str):
     # TODO: insert data too, and check in the tests below that we get that data back in the DataFrame
     #  https://github.com/objectiv/objectiv-analytics/issues/1093
+    if is_athena(engine):
+        with engine.connect() as conn:
+            conn.execute(f'drop table if exists {table_name};')
+            conn.execute(
+                (
+                    f'create external table {table_name}'
+                    '(a bigint, b string, c double, d date, e timestamp, f boolean) \n'
+                    f"location '{DB_ATHENA_LOCATION}';"
```

Comment:
Should we not have a location per test?
Maybe something like
```suggestion
                    f"location '{DB_ATHENA_LOCATION}/{table_name}';"
```

Of course this test doesn't actually write any data, so I guess this might not yet matter :thinking:

Reply:
huh that actually might help when avoiding multiple tests reading from the same location. will try

Reply:
Added here https://github.com/objectiv/objectiv-analytics/pull/1260/commits/c25e6d8f1b4eff48a9c887e52f8fce79ec9273f4